environments/package.json

```diff
@@ -3,11 +3,20 @@
   "name": "@aws-amplify/ui-environments",
   "version": "0.0.1",
   "scripts": {
+    "ai": "yarn --cwd ai/",
     "auth": "yarn --cwd auth/",
     "datastore": "yarn --cwd datastore/",
     "geo": "yarn --cwd geo/",
     "liveness": "yarn --cwd liveness/",
     "storage": "yarn --cwd storage/",
     "in-app-messaging": "yarn --cwd in-app-messaging/"
+  },
+  "devDependencies": {
+    "@aws-amplify/graphql-conversation-transformer": "raven",
+    "@aws-amplify/graphql-generation-transformer": "raven",
+    "@aws-amplify/backend-cli": "^1.1.1",
+    "aws-cdk": "^2.143.1",
+    "aws-cdk-lib": "^2.143.1",
+    "constructs": "^10.3.0"
```

Comment:
These are only necessary for now since we have to use the tagged release and override certain packages